### PR TITLE
Brackets.io Update: Please merge this pull request.

### PR DESCRIPTION
**Pull request by brackets-io**
Greetings from the [Brackets.io](https://brackets.io) & [phcode.dev](https://phcode.dev) community. We have [simplified extension development and publish workflows](https://github.com/phcode-dev/phoenix/wiki/How-To-Write-Extensions-And-Themes) for you.

## This Pull Request

Please merge in this pull request to be able to publish your extension/theme to the extension store directly from this repository. The pull request adds a [GitHub action](https://github.com/features/actions). The action is triggered whenever a GitHub release is created. The action notifies phcode extension services to pick up the newly released extension and publish it to Brackets/Phcode users.

https://registry.brackets.io/ is deprecated as you can directly publish your extension from GitHub.

# Publishing to the extension/theme store

**TLDR:** [Create a release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) in GitHub with an attached `extensions.zip` file to publish the extension to the extension/theme store.

Follow the below steps:

1. Clone your extension repo and increment the `version` field in `package.json` file before publishing.
1. Compress the extension folder into a zip file with name `extension.zip`.
   **NB: It is important to name the file as exactly `extension.zip`**
   image
1. Create a new release in GitHub and attach the above `extension.zip` file to
   the release.
    1. On GitHub.com, navigate to the main page of the repository.
    2. To the right of the list of files, click `Releases`.
       image
    3. Click on `Draft New Release`
       image
    4. Select `choose a tag`, type in your new extension version and click `Create new tag`. The tag name can be anything, but we recommend using the version number for clarity.
       image
    5. Fill out the remaining fields.
    6. Drag and drop the `extension.zip` file to the release.
       new extension
    7. Click on `Publish release` button.
        > See Release Example:
        > https://github.com/phcode-dev/extension-template/releases/tag/0.0.1
1. You will get an issue in your issue tab regarding the publishing status of the extension/theme in a few minutes.
    > See Example: https://github.com/phcode-dev/extension-template/issues/2
1. If there are any errors in publishing, please visit the link in the issue to see the errors. Fix and `retry` publishing the release.
   image
1. Once the extension is published, you should be able to see it in the
   extension store in https://phcode.dev
   image

**Please note that the extension will currently only be published to [phcode.dev](https://phcode.dev).**

To publish to the brackets extension store, please [raise an issue](https://github.com/brackets-cont/brackets/issues/new) after
publishing the extension to phcode.dev. This is a temporary solution till we have new native builds(read below).

**Please [see this link](https://github.com/phcode-dev/phoenix/wiki/How-To-Write-Extensions-And-Themes) for more documentation.**

## Creating new Extensions and Themes

Click on one of the below links and follow the instructions there to create new
themes and extensions:

-   [Create a theme](https://github.com/phcode-dev/theme-template)
-   [Create an extension](https://github.com/phcode-dev/extension-template)

## A Note on the future of Brackets/Phcode.dev

We are working hard to modernize [Brackets](https://github.com/brackets-cont/brackets/issues/112). New Windows/Linux/Mac native builds [are in the works](https://github.com/orgs/phcode-dev/projects/1) based on the web-first architecture of phcode.dev. Stay tuned for updates in the coming months:

1. Twitter: [phcode.dev](https://twitter.com/phcodedev) , [Brackets](https://twitter.com/BracketsCont)
1. [YouTube](https://www.youtube.com/@phoenixcodeeditor3764)
1. GitHub: [phcode.dev](https://github.com/phcode-dev/phoenix) , [Brackets](https://github.com/brackets-cont/brackets)

### CODE REVIEW

**Comment by asgerf** [comment]
Closing as I'm not maintaining this extension anymore
